### PR TITLE
Feature/22998 - Init package

### PR DESCRIPTION
**Pull request by rezkiy37**
### Details 
This is a base PR for a public package - `react-fast-pdf`. The goal here to have fundament configurations.
Since, it is a package, it includes 2 "projects". I mean the main part `./src` and the example `./example/**`. I used [Expensify/react-native-x-maps](https://github.com/Expensify/react-native-x-maps) like a template.

**Note**: In the future for the example, we should update link to `react-fast-pdf` in `./example/package.json`. Right now it uses fork [branch](https://github.com/rezkiy37/react-fast-pdf/tree/feature/22998-pdf-preview-library) to use current changes.

### Related Issues
https://github.com/Expensify/App/issues/22998, https://github.com/Expensify/App/issues/19918

### Manual Tests
1. Run `npm install` at the root.
2. Run `npm install` at the `example` directory.
3. Test all scripts in both `package.json`.

### Linked PRs
N/A

### CODE REVIEW

File: package.json
Comment (fabioh8010) [inline package.json:11]: Do we really need `npx` here?

File: package.json
Comment (fabioh8010) [inline package.json:13]: Do we really need `npx` here?

Comment (rezkiy37) [reply package.json]: Done - 618624c.

File: package.json
Comment (rezkiy37) [reply package.json]: Done - 618624c.

**Comment by rezkiy37** [comment]
> I don't see any checks running @rezkiy37. Do we need to do something in order to trigger those?

@allroundexperts, I can confirm that it triggers workflows, but on [my fork](https://github.com/rezkiy37/react-fast-pdf/actions). AFAIK, once we have workflows in the main repo, it is visible in PRs.

Screenshot 2023-08-16 at 13 43 45

Also, about "CLA Assistant" (`cla.yml`), we should configure GH secrets. How and who can do it, @allroundexperts?

**Comment by rezkiy37** [comment]
> Someone from our internal team would need to do that.

Do you know someone who can help us?

**Comment by AndrewGable** [comment]
It's set up as an org level secret, so you are good to go once it's merged 👍

**Comment by rezkiy37** [comment]
@allroundexperts, seems like all this CI jobs work once this PR has been merged. Any other blockers for review?

**Review by AndrewGable — APPROVED** [review]

**Pull request merged**
